package/src/components/TextInput/v1/TextInput.md

```diff
@@ -205,39 +189,27 @@ The default valid, invalid and clear icons can be overwritten by passing values
 
 These can also be provided through the main components context for your app.
 
-You can also provide the `iconClearAccessibilityText` prop with accessibility text for the clear icon.
-
-```jsx
-const TwoColumnExamples = require("../../../../../styleguide/src/components/TwoColumnExamples").default;
-
-const iconComponents = {
-  iconClear: <i className="fas fa-gavel" />,
-  iconError: <i className="fa fa-thumbs-down" />,
-  iconValid: <i className="fa fa-thumbs-up" />
-};
-
-<TwoColumnExamples hasDarkRightBackground>
-  <TextInput name="example" placeholder="Hint" value="Valid" hasBeenValidated components={iconComponents} />
-  <TextInput name="example" placeholder="Hint" value="Invalid" errors={["error"]} components={iconComponents} isOnDarkBackground />
-</TwoColumnExamples>
-```
 
 All of the icon props will accept a few different types of nodes to create the input icon.
  - React Component
  - `<i/>` tag used for icon fonts
  - `<svg>` tag for inline SVG
- - Plain text
+ - Plain text, Unicode symbols
 
-Inline SVG icon
+Here is how to use an SVG icon to replace the default `iconValid`, `iconError`, `iconClear` and `iconClearAccessibilityText`:
 
 ```jsx
 const TwoColumnExamples = require("../../../../../styleguide/src/components/TwoColumnExamples").default;
 
-const svg = <svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14" style={{ height: "100%" }}><path d="M9.926 9.105l-2.105-2.105 2.105-2.105-0.82-0.82-2.105 2.105-2.105-2.105-0.82 0.82 2.105 2.105-2.105 2.105 0.82 0.82 2.105-2.105 2.105 2.105zM7 1.176c3.227 0 5.824 2.598 5.824 5.824s-2.598 5.824-5.824 5.824-5.824-2.598-5.824-5.824 2.598-5.824 5.824-5.824z"></path></svg>;
+const iconComponents = {
+  iconClear: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"  style={{ height: "100%", verticalAlign: "middle" }}> <path fill="currentColor" d="M248 8C111 8 0 119 0 256s111 248 248 248 248-111 248-248S385 8 248 8zm0 448c-110.3 0-200-89.7-200-200S137.7 56 248 56s200 89.7 200 200-89.7 200-200 200zm-80-216c17.7 0 32-14.3 32-32s-14.3-32-32-32-32 14.3-32 32 14.3 32 32 32zm160 0c17.7 0 32-14.3 32-32s-14.3-32-32-32-32 14.3-32 32 14.3 32 32 32zm4 72.6c-20.8 25-51.5 39.4-84 39.4s-63.2-14.3-84-39.4c-8.5-10.2-23.7-11.5-33.8-3.1-10.2 8.5-11.5 23.6-3.1 33.8 30 36 74.1 56.6 120.9 56.6s90.9-20.6 120.9-56.6c8.5-10.2 7.1-25.3-3.1-33.8-10.1-8.4-25.3-7.1-33.8 3.1z" /></svg>,
+  iconError: "X",
+  iconValid: "\u2714"
+};
 
```

Comment:
@aldeed Is this how someone would add their own custom SVG? Or should this `iconClear` be saved in `appComponents.js`?

Reply:
@machikoyasuda If they want a custom clear icon everywhere, they should provide it in their components context. If they want it in just a single place in the app or just a few, they should pass it in those places only, like this example. Not sure if you want to mention the global context way here, but you could. Once people understand how the context and the `components` prop are related, it works the same everywhere. So we wouldn't have to explain it in every component docs, but it also wouldn't hurt.